java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -73,6 +82,19 @@ public boolean equals(Object obj) {
       }
       return false;
     }
+
+    @Override
+    public String toString() {
+      String tree = i<block.elements().size()?""+block.elements().get(i).kind()+block.elements().get(i).firstToken().line():"";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS864)